src/Consolonia.NUnit/UnitTestConsole.cs

```diff
@@ -170,6 +172,41 @@ public async Task KeyInput(Key key, RawInputModifiers modifiers = RawInputModifi
             await WaitRendered().ConfigureAwait(true);
         }
 
+        public IInputElement GetFocus()
+        {
+            return _lifetime.MainWindow.FocusManager.GetFocusedElement();
+        }
+
+        public async Task SetFocus<T>(string controlName)
```

Comment:
Not used

Reply:
yeah, ok, I can do a GetFocus().Focus();